kod/object/passive/moon.kod

```diff
@@ -114,7 +114,7 @@ messages:
    }
 
    AddBackgroundObject(who = $)
-   "Called by user upon logon"
+   "Called by user upon logon and after system save"
```

Comment:
Change the comment to say what the function does, not when it is called.  This is a perfect illustration of why not to document when it's called.